products.js

```diff
@@ -21,7 +21,7 @@ const {Annealer} = require('fathom-web/optimizers');
 const {staticDom} = require('fathom-web/utils');
 const tuningRoutines = {
                         'title' : {'routine': tunedTitleFnodes, 'coeffs': []},
-                        // 'price' : {'routine': tunedPriceFnodes, 'coeffs': [1, 1, 0.5]},
+                        'price' : {'routine': tunedPriceFnodes, 'coeffs':   [ 1.6, 1, 0.7, 0.7, 1.9000000000000001, 0.39999999999999997, 1, 1.3, 0.7, 1.3, 1.3, 1.3, 0.09999999999999998, 1, 0.39999999999999997, 0.39999999999999997, 0.7, 1.9000000000000001, 7.3 ]},
```

Comment:
A lot of these numbers are probably equal to their shorter representations, like 1.9 and 3.4. Floating point has only so much resolution, so you tend to get artifacts like this. (After all, your tuning routine adds or subtracts only tenths at a time, so how could it be otherwise?) We can thus shorten the spelling here for the poor humans who get to read it. :-)